sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -476,7 +477,16 @@ def _add_argparse_args(cls, parser):
                         action='store_true',
                         help='Update an existing streaming Cloud Dataflow job. '
                         'Experimental. '
-                        'See https://cloud.google.com/dataflow/pipelines/'
+                        'See https://cloud.google.com/dataflow/docs/guides/'
+                        'updating-a-pipeline')
+    parser.add_argument('--transform_name_mapping',
```

Comment:
As far as I can tell python does not have a place for streaming specific options.  Is it worth making it and moving this there?

Reply:
It's probably worth adding generic streaming options, I think this particular option is more dataflow oriented.

Reply:
This option is probably more update related, which is currently only support with dataflow runner I guess?

Reply:
That is correct. Update is only supported by Dataflow.